src/v2/util/Animations.js

```diff
@@ -0,0 +1,146 @@
+/*!
```

Comment:
if it's identical, I think we shall just reference to `src/util/Animation` instead of copying.
same idea with a few other util files.